src/scenes/TaskDetails/index.tsx

```diff
@@ -0,0 +1,73 @@
+import {Profile} from "../../types/Profile";
+import {useContext} from "react";
+import userContext from "../../types/UserContext";
+import {Badge, Button, Card} from "flowbite-react";
+import {HiCheck, HiPencil, HiTrash} from "react-icons/hi";
+
+
+interface TaskViewProps {
+    isAgentView: boolean,
+  }
+
+function TaskDetails({ isAgentView }: TaskViewProps){
```

Comment:
```suggestion
function TaskDetailsView({ isAgentView }: TaskViewProps){
```
Lets rename this as `TaskDetailsView` to keep the naming convention consistent, make sure you rename the directory name as well, (hint: use refactor on IntelliJ)

Reply:
All the instances of TaskViewProps and TaskDetails were renamed to TaskDetailsViewProps and TaskDetailsView respectively.